openapi/services/journals.yaml

```diff
@@ -0,0 +1,129 @@
+Summary:
+  get:
+    summary: Summary information about a publication indexed by ADS
+    parameters:
+      - $ref: '../parameters.yaml#/JournalBibstem'
+    description: >
```

Comment:
This param is case sensitive (when testing) - if that's as it is meant to be, just add a note about that (either here or in the field descriptor: `openapi/parameters.yaml`, line 230 in this PR)

Reply:
I've added a "(Case-sensitive)" note to parameters.yaml#/JournalBibstem.  I also added the example "ApJS"